test/Oscoin/Test/Crypto/Blockchain/Block/Helper.hs

```diff
@@ -0,0 +1,15 @@
+module Oscoin.Test.Crypto.Blockchain.Block.Helper where
+
+import           Oscoin.Prelude
+
+import           Oscoin.Crypto.Hash (HasHashing)
+import           Oscoin.Crypto.PubKey (HasDigitalSignature, PublicKey)
+
+import           System.IO.Unsafe (unsafePerformIO)
+import           Test.QuickCheck
+
+-- | Generates an arbitrary beneficiary.
+defaultBeneficiary
+    :: (Arbitrary (PublicKey c), HasDigitalSignature c, HasHashing c) => PublicKey c
+defaultBeneficiary = unsafePerformIO $ generate arbitrary
+{-# NOINLINE defaultBeneficiary #-}
```

Comment:
This seemed like the least intrusive way to add the beneficiary to the tests for now. When we implement a "burn" address, we can switch it to that, and remove the unsafe code.